README.md

```diff
@@ -28,24 +28,29 @@ The current multibase table is [here](multibase.csv):
 
 ```
 encoding      codes   name
-base1         1       unary tends to be 11111
-base2         0       binary has 1 and 0
-base8         7       highest char in octal
-base10        9       highest char in decimal
-base16        F, f    highest char in hex
-base32        U, u    rfc4648 - highest letter
-base32hex     V, v    rfc4648 - highest char
-base58flickr  Z       highest char
-base58btc     z       highest char
-base64        y       rfc4648 highest char
-base64url     Y       rfc4648 highest char
+base1,        1,    unary tends to be 11111
+base2,        0,    binary has 1 and 0
+base8,        7,    highest char in octal
+base10,       9,    highest char in decimal
+base16,       F, f, highest char in hex
+base32hex,    V, v, rfc4648 no padding - highest char
+base32hexpad, T, t, rfc4648 with padding
+base32,       B, b, rfc4648 no padding
+base32pad,    C, c, rfc4648 with padding
+base32z,      h,    z-base-32 - used by Tahoe-LAFS - highest letter
+base58flickr, Z,    highest letter
+base58btc,    z,    highest letter
+base64,       m,    rfc4648 no padding
+base64pad,    M,    rfc4648 with padding - MIME encoding
+base64url,    u,    rfc4648 no padding
+base64urlpad, U,    rfc4648 with padding
+binary,       X,    8-bit binary (encoder and decoder keeps data unmodified)
 ```
 
 These encodings are being considered:
 
 ```
 base128
-base256ascii  X       ascii
```

Comment:
i believe the intent of this encoding was more complicated than you realize. that it means is "ascii values", meaning it's a base256 _value_[0] from the ascii table. Functionally, because _we_ use ascii-compatible systems, it's almost the same as binary, but it's _not_ the same, as not all architectures have used the ascii table, and there are other encodings. This is definitely a confusing encoding though so we should probably drop it from the readme, i vaguely recall a useful reason for it, but i'm not recalling it now.

[0] so, a byte, which happens to be the same as base2 bytes (binary bytes) because of conventions.